Source/TableEditingController.swift

```diff
@@ -34,7 +34,7 @@ public struct TableEditingController {
 
      - returns: `true` if the specified row is editable, `false` otherwise.
      */
-    public typealias CanEditRowConfig = (_ tableView: UITableView, _ indexPath: IndexPath) -> Bool
+    public typealias CanEditRowConfig = (_ tableView: UITableView, _ indexPath: IndexPath, _ dataSource: inout DataSource) -> Bool
```

Comment:
ah, you mean specifically in the `canEditRow:` closure?

we definitely need the dataSource reference in the `commitEditing:` closure.

yeah -- maybe we only pass in `Item` for this one.